tools/rostest/src/rostest/rostest_parent.py

```diff
@@ -35,6 +35,7 @@
 import logging
 import sys
 
+import rosgraph
```

Comment:
Should this translate to an explicit dependency on `rosgraph`? It doesn't currently depend directly on it:

https://github.com/ros/ros_comm/blob/indigo-devel/tools/rostest/package.xml